<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>p8microclimate_heat_gui</class>
 <widget class="QDialog" name="p8microclimate_heat_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>444</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Gridsize [m]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="le_landuse"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Shapefile</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Landuse Map</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pb_map">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pb_shape">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Percentile</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="cmb_perc">
      <item>
       <property name="text">
        <string>Median - Stdev (i.e. 20%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Median (i.e. 50%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Median + Stdev (i.e. 80%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sample Median +- Stdev</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="le_shape"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pb_landuse">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="le_gridsize"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="le_map"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Map Picture</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pb_placeTech">
      <property name="text">
       <string>Place Technologies / View Temperature</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Technology File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="le_techFile"/>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pb_techFile">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bBox</sender>
   <signal>accepted()</signal>
   <receiver>p8microclimate_heat_gui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBox</sender>
   <signal>rejected()</signal>
   <receiver>p8microclimate_heat_gui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
